modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/build.gradle

```diff
@@ -38,9 +38,12 @@ dependencies {
 	testIntegrationCompile project(":apps:data-engine:data-engine-rest-api")
 	testIntegrationCompile project(":apps:dynamic-data-mapping:dynamic-data-mapping-form-builder")
 	testIntegrationCompile project(":apps:export-import:export-import-test-util")
+	testIntegrationCompile project(":apps:fragment:fragment-api")
```

Comment:
@alberttgomes, please update commit message to be like:

Implement test to remove declared browser sniffer variables